packages/runtime/datastore/src/channelStorageService.ts

```diff
@@ -17,14 +17,12 @@ export class ChannelStorageService implements IChannelStorageService {
 		tree: ISnapshotTree,
 		results: { [path: string]: string },
 	) {
-		// eslint-disable-next-line guard-for-in, no-restricted-syntax
-		for (const path in tree.trees) {
-			ChannelStorageService.flattenTree(`${base}${path}/`, tree.trees[path], results);
+		for (const [path, subtree] of Object.entries(tree.trees)) {
```

Comment:
I can never keep the differences between for-in and for-of straight... just need to triple check this doesn't change the underlying logic unexpectedly.